vimeo-networking/src/main/java/com/vimeo/networking/interceptors/CacheControlInterceptor.java

```diff
@@ -33,16 +33,16 @@
 
 /**
  * Rewrite the server's cache-control header because our server sets all {@code Cache-Control} headers
- * to {@code no-store}
- * Created by brentwatson on 8/7/17.
+ * to {@code no-store}.
+ *
  */
 public class CacheControlInterceptor implements Interceptor {
 
     @Override
     public Response intercept(Chain chain) throws IOException {
         return chain.proceed(chain.request())
                 .newBuilder()
-                .header(Vimeo.HEADER_CACHE_CONTROL, Vimeo.HEADER_CACHE_PUBLIC)
+                .header(Vimeo.HEADER_CACHE_CONTROL, Vimeo.HEADER_CACHE_PUBLIC_WITH_MAX_AGE)
```

Comment:
Rather than hardcoding the cache age, should we be using the value available on the configuration object via `Configuration.getCacheMaxAge()`?

Reply:
Yes. I made the change.